pipes/rules/assembly.rules

```diff
@@ -49,12 +50,36 @@ rule assemble_trinity:
         UGER        = config.get('UGER_queues', {}).get('short', '-l h_rt 04:00:00'),
         n_reads     = str(config["trinity_n_reads"]),
         logid       = "{sample}",
-        subsamp_bam = config["tmp_dir"]+'/'+config["subdirs"]["assembly"]+'/{sample}.subsamp.bam'
     run:
         makedirs(expand("{dir}/{subdir}",
             dir=[config["data_dir"],config["tmp_dir"]],
             subdir=config["subdirs"]["assembly"]))
-        shell("{config[bin_dir]}/assembly.py assemble_trinity {input.bam} {input.clipDb} {output.fasta} --n_reads={params.n_reads} --outReads {params.subsamp_bam} --threads {resources.threads} --JVMmemory 5g")
+        shell("{config[bin_dir]}/assembly.py assemble_trinity {input.bam} {input.clipDb} {output.fasta} --n_reads={params.n_reads} --outReads {output.subsamp_bam} --threads {resources.threads} --JVMmemory 5g")
+
+rule assemble_spades:
```

Comment:
Are you convinced that you would rather deploy SPAdes as a post-Trinity polish step instead of as an independent alternative to Trinity altogether?